about.css

```diff
@@ -1,6 +1,10 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
 @import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css");
 
+html {
+  scroll-behavior: smooth;
+}
+
 body {
```

Comment:
There's a horizontal scroll in both versions that makes for bad UI/UX. I'm using Google Chrome for reference.

<img width="230" alt="Screen Shot 2022-07-27 at 9 04 00" src="https://user-images.githubusercontent.com/82426682/181267164-fe05574c-271e-4e96-87f4-56494b639442.png">

Please fix it.